BrowserKit/Tests/WebEngineTests/WKContentScriptManagerTests.swift

```diff
@@ -41,7 +41,7 @@ final class WKContentScriptManagerTests: XCTestCase {
                                  forSession: session)
 
         XCTAssertEqual(script.scriptMessageHandlerNamesCalled, 1)
-        let config = session.webView.engineConfiguration as! MockWKEngineConfiguration
+        guard let config = session.webView.engineConfiguration as? MockWKEngineConfiguration else { return }
```

Comment:
In these situations where we fail to cast to an expected type, instead of just returning let's also add before the return statement `XCTFail` with a nice message so we know exactly what happened. This comment applies to the cases below too.